hepcrawl/spiders/arxiv_spider.py

```diff
@@ -100,6 +100,10 @@ def parse_node(self, response, node):
             license_url=node.xpath('.//license//text()').extract_first()
         )
         record.add_value('license', license)
+        record.add_value(
+            'copyright_material',
+            'preprint',
+        )
```

Comment:
I don't think that we should be adding this to the spider, unless we add the full copyright (at least holder or statement).

@michamos wdyt?